scripts/readme-upload-missing-images.ts

```diff
@@ -0,0 +1,104 @@
+import * as fsPromises from "fs/promises";
+import * as fs from "fs";
+import path from "path";
+import axios from "axios";
+import FormData from "form-data";
+
+const readmeUploadMissingImages = async () => {
+  if (process.env.README_IO_AUTH?.length < 10) {
+    console.log("`README_IO_AUTH` was not provided in `.env` file :/");
+    return;
+  }
+  const basePath = path.join(__dirname, "../docs");
+  const pathsToFiles = await getFiles(basePath);
+  for (const pathToFile of pathsToFiles) {
```

Comment:
imo this script is a bit difficult to read.. 
In the future, if we find free time, I would consider separating more functions, e.g. one that processes a file, etc.
maybe create const for `process.env.README_IO_AUTH` etc.
or even adding descriptive comments to make fixing/editing the script easier

Reply:
added comments, thanks